sputnikdao2/src/views.rs

```diff
@@ -41,7 +41,7 @@ impl Contract {
 
     /// Returns staking contract if available. Otherwise returns empty.
     pub fn get_staking_contract(&self) -> AccountId {
-        self.staking_id.clone().unwrap_or_default()
+        self.staking_id.clone().unwrap()
```

Comment:
This will have different behaviour than what previously was there. I don't think the pattern that existed before was good, but if you want to match exactly you can do something like this:
```rust
    pub fn get_staking_contract(self) -> String {
        self.staking_id.map(String::from).unwrap_or_default()
```

Reply:
switched!